src/consumers/index.js

```diff
@@ -52,4 +48,6 @@ exports.initialize = async function () {
   // managedConsumer.on('consumer-restart-failed', (err) => {
   //  // handle restart failure
   // })
+
+  return { eventConsumer, workConsumer }
```

Comment:
exported these so that we can test against the actual consumers we create when running the service.